src/app/public/modules/timepicker/timepicker.directive.ts

```diff
@@ -66,9 +66,19 @@ export class SkyTimepickerInputDirective implements
   public pickerChangedSubscription: Subscription;
   private _timeFormat: string = 'hh';
 
+  /**
+   * Creates the timepicker input field and picker. Place this attribute on an `input` element,
+   * and wrap the input in a `sky-timepicker` component.
+   * This attribute must be set to the instance of the `sky-timepicker`.
+   * @required
+   */
   @Input()
   public skyTimepickerInput: SkyTimepickerComponent;
 
+  /**
+   * Specifies the 12-hour hh or 24-hour HH time format for the input.
```

Comment:
Based on the existing docs, I believe `hh` and `HH` should be code snippets.

Reply:
Done.